### ISSUE TITLE
android.nfc.NfcAdapter.isEnabled() - null object reference

### ISSUE THREAD

**Issue by calebmackdavenport** [stripe/stripe-react-native#1306]
**Describe the bug**

We use bugsnag for crashlytics and error reporting.
Have seen a handful of instances of this error:
![image](https://user-images.githubusercontent.com/30021449/219907034-f5c31c14-1335-48ca-9b4d-22f6dc2a1f65.png)

We are currently on:
`"@stripe/stripe-react-native": "0.22.0",`

Here is a stack trace of the error:
```
java.lang.NullPointerException: Attempt to invoke virtual method 'boolean android.nfc.NfcAdapter.isEnabled()' on a null object reference
        at com.reactnativestripesdk.pushprovisioning.PushProvisioningProxy.isNFCEnabled(PushProvisioningProxy.kt:36)
        at com.reactnativestripesdk.StripeSdkModule.canAddCardToWallet(StripeSdkModule.kt:680)
        at java.lang.reflect.Method.invoke(Method.java:-2)
        at com.facebook.react.bridge.JavaMethodWrapper.invoke(JavaMethodWrapper.java:372)
        at com.facebook.react.bridge.JavaModuleWrapper.invoke(JavaModuleWrapper.java:188)
        at com.facebook.jni.NativeRunnable.run(NativeRunnable.java:-2)
        at android.os.Handler.handleCallback(Handler.java:942)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at com.facebook.react.bridge.queue.MessageQueueThreadHandler.dispatchMessage(MessageQueueThreadHandler.java:27)
        at android.os.Looper.loopOnce(Looper.java:240)
        at android.os.Looper.loop(Looper.java:351)
        at com.facebook.react.bridge.queue.MessageQueueThreadImpl$4.run(MessageQueueThreadImpl.java:228)
        at java.lang.Thread.run(Thread.java:1012)
```

**To Reproduce**
I have not successfully recreated the error and am not fully certain how it is presenting, other than following a call to canAddCardToWallet on a handful of android devices.

**Expected behavior**
I am uneducated on android fundamentals. I don't pretend to know anything, but I am guessing that this is crashing instead of presenting a `null` for some reason, as is suggested in the [documentation](https://developer.android.com/reference/android/nfc/NfcAdapter#getDefaultAdapter(android.content.Context)).

**Smartphone (please complete the following information):**
 - Device List:
![image](https://user-images.githubusercontent.com/30021449/219907188-9a2fd522-4ae8-4e78-827a-6e6f2ceeda35.png)
 - OS: Android
 - Version: Primarily Android 13, but impacted versions reported:
_I would place very little weight on 7 and 9, but that's the list_
![image](https://user-images.githubusercontent.com/30021449/219907209-6a7d551e-3095-4137-b133-15c403cd86f3.png)

**Additional context**
Happy to trial a new version if that is recommended. I see there are several notes regarding `canAddCardToWallet` in the changelog. But still felt this warranted raising an issue. If you need more information, I am actively working on gathering it and will hopefully update with specific reproduction steps.

**Comment by charliecruzan-stripe** [comment]
hey @calebmackdavenport , got a fix incoming for this, thanks for the report!

**Comment by charliecruzan-stripe** [comment]
published in 0.25.0 👍
